source/vocab/relations.ttl

```diff
@@ -404,10 +404,10 @@
     owl:equivalentProperty foaf:isPrimaryTopicOf .
 
 :language a owl:ObjectProperty;
-    rdfs:label "language"@en, "språk"@sv;
+    rdfs:label "språk"@sv;
     # TODO worklanguage? 
     rdfs:comment "Språk förknippat med en resurs eller dess delar."@sv;
-    sdo:domainIncludes :Creation, :StructuredValue;
+    sdo:domainIncludes :Creation, :Summary, :TableOfContents, :Title, marc:Libretto, marc:SubtitlesOrCaptions, marc:SungOrSpokenText;
```

Comment:
I'm wondering if we want an (abstract) subclass of StructuredValue as a base class for some of these? Possibly a base for Note as well, which reasonably can have a language. I'd call it `Note`, but that's obviously taken... Perhaps `Annotation`, to bring back an old BF1-term...

Reply:
I think we first should figure out what the Marc classes actually are about and their place in the model, then I need to remove Title which already is a StructuredValue :)

Reply:
Never mind that brainmelt, I removed SV to fix this :)